packages/twenty-front/src/modules/views/components/ViewBarDetails.tsx

```diff
@@ -68,8 +64,10 @@ const StyledCancelButton = styled.button`
 `;
 
 const StyledFilterContainer = styled.div`
-  align-items: center;
   display: flex;
+  align-items: center;
+  flex: 1;
+  overflow-x: hidden;
```

Comment:
we need item to stack

Reply:
On the dev environment, filters stack on each other in column manner. But in figma they are stacking row wise and having a scroll. Is column stack behaviour we need to follow ?